helpers/utils.js

```diff
@@ -1,5 +1,17 @@
 function getRandomIndex(array) {
-    return Math.floor(Math.random() * array.length);
+  return Math.floor(Math.random() * array.length);
 }
 
-exports.getRandomIndex = getRandomIndex;
\ No newline at end of file
+exports.getRandomIndex = getRandomIndex;
+
+/* Hi your getRandomIndex method is not working
```

Comment:
Would you be able to explain which part of the `getRandomMethod` function is not working? I've ran it multiple times and it seems to return back a random number every time.

Reply:
Hi, I'm sorry. getrandomMethod is working totally fine. the issue was happen when exporting the lists.
It export as a list in a object. So we should fetch the list inside the object by calling dessets.desserts
I think that we can put it as another issue? what would you suggest

![screenshot_1](https://user-images.githubusercontent.com/49401137/96531864-3a7cdf80-12a8-11eb-9796-4a1a6d1020da.png)

The right way

![screenshot](https://user-images.githubusercontent.com/49401137/96531600-a3b02300-12a7-11eb-89d8-a751d2a5b0d7.png)

Reply:
This PR has been closed, but what you could do is probably

`const { desserts } = require("./data/desserts_list")